RHEL/6/input/system/permissions/files.xml

```diff
@@ -432,7 +432,7 @@ unprivileged users to elevate privileges. The presence of these files should be
 strictly controlled on the system.</rationale>
 <ident cce="26497-8" />
 <oval id="file_permissions_unauthorized_suid" />
-<ref nist=""/>
+<ref nist="AC-6"/>
```

Comment:
AC-6 is the broader "least privilege" requirement. Let's refine this to AC-6(1), which reads:

````
AC-6(1)	
LEAST PRIVILEGE | AUTHORIZE ACCESS TO SECURITY FUNCTIONS
The organization explicitly authorizes access to [Assignment: organization-defined security functions (deployed in hardware, software, and firmware) and security-relevant information].
````

Arguably, SUID's are "security-defined software"